src/settings-ui/Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -68,6 +68,7 @@
                         IsTabStop="True"
                         IsOpen="{x:Bind Mode=OneWay, Path=ViewModel.UseLegacyPressWinKeyBehavior}"
                         IsClosable="False"
+                        IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"
```

Comment:
Isn't this fixed. I think I opened an issue in the past.

Reply:
Not yet
![image](https://user-images.githubusercontent.com/65828559/180643105-f3ed1621-7607-4a66-9845-04c4d08600c0.png)